public/index.html

```diff
@@ -5,8 +5,6 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <meta http-equiv="X-UA-Compatible" content="ie=edge" />
   <title>Authors Haven</title>
-  <meta name="viewport" content="width=device-width, initial-scale=1" />
```

Comment:
Hey Daniel, why did you remove the viewport tag?

Reply:
@Chrismarcel it is already in line 5. No need to include it second time.

Reply:
Alright @danprocoder. Thanks for the clarity